doc/website/index.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: splash
+permalink: /
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/mm-home-page-feature.jpg
+  actions:
+    - label: "<i class='fas fa-download'></i> Install now"
+      url: "/get-started/setup/"
+excerpt: >
+  Develop and design Hardware seamlessly with the power of ROHD.<br />
+  <small><a href="https://github.com/intel/rohd/releases">Latest release v0.4.1</a></small>
+feature_row:
+  - image_path: /assets/images/mm-customizable-feature.png
+    alt: "Flexible Development"
+    title: "Flexible Development"
+    excerpt: "The development allows run-time dynamic module port definitions (numbers, names, widths, etc.) and internal module logic, including recursive module contents. Conversion of modules to equivalent, human-readable, structurally similar SystemVerilog for integration or downstream tool consumption"
+    
+  - image_path: /assets/images/mm-responsive-feature.png
+    alt: "Modern Language"
+    title: "Modern Language"
+    excerpt: "Dart programming language enable fewer line of code in hardware design and verification. Modern IDE like VSCode also provides excellent static analysis, autocomplete, debugger, git, lint and etc which enables quality development."
+
+  - image_path: /assets/images/mm-free-feature.png
+    alt: "Open Source Project"
+    title: "Open Source Project"
+    excerpt: "ROHD is simple and fast, with no cumbersome build systems or EDA vendor tools. Open source community is established, allowing developers to contribute or extend the framework while obtaining assistance from the community world wide."
+      
+feature_row2:
+  - image_path: /assets/images/abstract_layer.jpg
```

Comment:
Where did these images come from?  We need to attribute them depending on the license

Reply:
<!-- Unsplash Image Source: https://unsplash.com/photos/FVgECvTjlBQ, https://unsplash.com/photos/KuCGlBXjH_o -->


Added this line of code to indicate image source